react/features/base/icons/svg/index.ts

```diff
@@ -47,7 +47,6 @@ export { default as IconHelp } from './help.svg';
 export { default as IconHighlight } from './highlight.svg';
 export { default as IconImage } from './image.svg';
 export { default as IconInfoCircle } from './info-circle.svg';
-export { default as IconBurger } from './burger.svg';
```

Comment:
Are we using the icon at all? If not, can we delete it?

Reply:
The icon is not being used, that's why I deleted it.

Reply:
Cool! By delete i meant the SVG too.

Reply:
Oh, ok. You are talking about the actual file. I will delete it, yes.